docs/content-services/components/search-filter-tabbed.component.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: Search filter tabbed component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-13
+---
+
+# [Search filter tabbed component](../../../lib/content-services/src/lib/search/components/search-filter-tabbed/search-filter-tabbed.component.ts "Defined in search-filter-tabbed.component.ts")
+
+Represents a container [search widget](../../../lib/content-services/src/lib/search/models/search-widget.interface.ts)
```

Comment:
Is that correct? 
SearchFilterTabbedComponent does not implement SearchWidget interface: 
<img width="666" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/cfaf6c60-ed11-4a2e-b360-60a600e0642a">

SearchDateRangeAdvancedTabbedComponent implements SearchWidget 
<img width="953" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/6d424ca3-900a-4a35-b017-6296b7574617">

Reply:
Removed reference to SearchWidget. This was the case before, when we were planning to create the tabbed component as its own, reusable widget. But that approach has since been scrapped. This was probably a leftover line from that. The new documentation now uses the following language - 

`Represents a container for [Search Filters](file:///home/swapnil.verma/workspaces/alfresco-frontend/alfresco-ng2-components/docs/content-services/components/search-filter.component.md) to provide a tabbed user interface for the filters.`